mysql-test/suite/rocksdb/t/bulk_load_unsorted_errors.test

```diff
@@ -0,0 +1,8 @@
+--source include/have_rocksdb.inc
+--source include/have_nodebug.inc
```

Comment:
Why?

Reply:
Not sure, came from upstream. I suspect though it was due to many debug assertions that were present in bulk load functionality that would be triggered by the test. Many of these have been removed by one of my submitted patches and replaced with client error propagation.

Reply:
OK, I see the problem here. This functionality seems broken. There is a 'set' callback for the rocksdb_bulk_load_allow_unsorted. This callback calls my_error, which is not really a valid thing to do. When cleaning up the statement execution, debug builds assert if the status has already been set :

 375 void Diagnostics_area::set_ok_status(ulonglong affected_rows,
 376                                      ulonglong last_insert_id,
 377                                      const char *message_text)
 378 {
 379   DBUG_ENTER("set_ok_status");
 380   DBUG_ASSERT(! is_set());

So this is an upstream bug. They should be using a 'check' callback for the sysvar instead. I will make up a patch for this and send it upstream.

Reply:
See https://github.com/facebook/mysql-5.6/pull/755